### PR TITLE
Login, Logout, Signup, and Home page

### PR DESCRIPTION
**Pull request by shivangipathak01**
This branch contains the backend and frontend changes made within Sprint 1 to implement the overall functionality for the Login, Logout, Signup, and Home page.

### CODE REVIEW

File: backend/controllers/user.js
Comment (manushreesaboo) [inline backend/controllers/user.js:1]: mostly formatting changed

File: backend/routes/user.js
Comment (manushreesaboo) [inline backend/routes/user.js:1]: mostly formatting changed

**Review by manushreesaboo — COMMENTED** [review]
looks good to me (frontend wise). Backend mostly formatting changed, and Backend/index.js had some additional imports and checks

**Comment by HarshGajabi** [comment]
Looks good to me!

**Review by HarshGajabi — APPROVED** [review]

**Pull request merged**